README.md

```diff
@@ -34,6 +34,7 @@ I aim to update this list as new models are released, but please submit a pull r
 | **[Virchow 2](https://arxiv.org/abs/2408.00738)**                                                                                                 | [Paige](https://paige.ai) / [MSR](https://www.microsoft.com/en-us/research/lab/microsoft-health-futures) | :x:                | Aug 2024[\*](https://arxiv.org/abs/2408.00738v1)                                                      | DINOv2 (+ ECT and KDE)                                                          | **3.1M**                        | 2B       | 230K       | 4096       |            | ViT-H with 4 registers | 632M       | 3584      | 224        | proprietary (from MSKCC and international sites) |                                                                                                                                                                                                                                                                                                                               |
 | **[Virchow 2G](https://arxiv.org/abs/2408.00738)**                                                                                                | [Paige](https://paige.ai) / [MSR](https://www.microsoft.com/en-us/research/lab/microsoft-health-futures) | :x:                | Aug 2024[\*](https://arxiv.org/abs/2408.00738v1)                                                      | DINOv2 (+ ECT and KDE)                                                          | **3.1M**                        | 2B       | 230K       | 3072       |            | ViT-G with 8 registers | 1.9B       | 3584      | 224        | proprietary (from MSKCC and international sites) |                                                                                                                                                                                                                                                                                                                               |
+| [Phikon-v2](https://arxiv.org/abs/2409.09173v1)                                                                                                   | [Owkin](https://www.owkin.com)                                                                           | :white_check_mark: | Sep 2024[\*](https://arxiv.org/abs/2409.09173v1)                                                      | DINOv2                                                                          | 58.4k                           | 456M     |            | 4096       | 250K       | ViT-L                  | 307M       | 1024      | 224        | TCGA, CPTAC, GTEx                                | [<img src="https://huggingface.co/datasets/huggingface/brand-assets/resolve/main/hf-logo.svg" width="25">](https://huggingface.co/owkin/phikon-v2)                                                                                                                                                                            |
```

Comment:
nit: 
Can you remove `v1` from the leftmost Phikon link, to ensure it always points to the newest version?
Also, I think that the dataset is a bit more than TCGA, CPTAC, GTEx. In my opinion `PANCAN-XL (TCGA, CPTAC, GTEx, proprietary)` would describe the dataset more accurately.